spec/replication/util/migration_order_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe "migration order" do
+  let(:current_release_migrations) do
+    File.read(File.join(__dir__, 'data/darga_migrations')).split.map(&:to_i)
```

Comment:
I know this file is generated by us but maybe we should do a `.sort` so the file doesn't have to be sorted.